### PR TITLE
#161913720 Display article reading time and arrange articles in descending order

### PR DESCRIPTION
**Pull request by mendozabree**
#### What does this PR do?

Allow all users to view articles with reading time and from the newest articles to the oldest

#### Description of Task to be completed?

Currently, the articles returned display `null` for reading time and are in ascending order(oldest to latest)

This task allows users view article reading time and the articles are displayed in descending order

#### How should this be manually tested?

- Run the application, `./manage.py runserver`
- Navigate to the endpoint to view all articles *GET* `http://localhost:8000/api/article/`

#### What are the relevant pivotal tracker stories?

[#161913720](https://www.pivotaltracker.com/story/show/161913720)

#### Any background context you want to add?
N/A

#### Screenshots
image

### CODE REVIEW

**Review by samuelbwambale — COMMENTED** [review]
@mendozabree The functionality is okay. However, the social login tests are failing. Kindly consult @kakaemma for help on that

**Review by MuhanguziDavid — COMMENTED** [review]
Nice work @mendozabree . However, please ensure that the tests pass.

**Comment by mendozabree** [comment]
@samuelbwambale @MuhanguziDavid thank you, I have gotten in touch with Emma.

File: authors/apps/articles/views.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def create(self, request):
         serializer_data = request.data.get('article', {})
 
         time_to_read = Article.article_reading_time(serializer_data['body'])
-        serializer_data['readingTime'] = time_to_read
+        serializer_data['reading-time'] = time_to_read
```
Comment (RonKbS) [inline authors/apps/articles/views.py]: Good job fixing this @mendozabree.
This should be an underscore, not a dash.
Please check to make sure.

Comment (mendozabree) [reply authors/apps/articles/views.py]: Checked it out, thanks @RonKbS you are right it is supposed to be an `underscore` let me change that.

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

**Review by kakaemma — APPROVED** [review]
LGTM

**Review by RonKbS — APPROVED** [review]
LGTM

**Pull request merged**